worlds/hk/__init__.py

```diff
@@ -166,29 +167,29 @@ def __init__(self, world, player):
         self.vanilla_shop_costs = deepcopy(vanilla_shop_costs)
 
     def generate_early(self):
-        world = self.multiworld
-        charm_costs = world.RandomCharmCosts[self.player].get_costs(world.random)
-        self.charm_costs = world.PlandoCharmCosts[self.player].get_costs(charm_costs)
-        # world.exclude_locations[self.player].value.update(white_palace_locations)
+        options = self.options
+        charm_costs = options.RandomCharmCosts.get_costs(self.random)
+        self.charm_costs = options.PlandoCharmCosts.get_costs(charm_costs)
+        # self.multiworld.exclude_locations[self.player].value.update(white_palace_locations)
```

Comment:
Might as well fully update the comment
```suggestion
        # options.exclude_locations.value.update(white_palace_locations)
```

Reply:
i updated the comment for world: multiworld because that's confusing
but i agree, might as well update it to the new format too